ch_pipeline/core/telescope.py

```diff
@@ -196,12 +199,14 @@ def _finalise_config(self):
     @property
     def fwhm_e(self):
```

Comment:
You should probably use `caput.cache.cahed_property` as the decorator here to avoid redoing the polynomial fit every time it is referenced.

Reply:
Thanks, I will have a look at it.